app/views/ErrorTemplateNoauth.scala.html

```diff
@@ -16,18 +16,18 @@
 
 @import scala.concurrent.ExecutionContext
 
-@this(mainTemplate: MainTemplate)
+@this(mainTemplate: MainTempleteNew)
 
 @(pageTitle: String, heading: String, message1: String, additionalMessages: List[String])(implicit request: Request[_],
         messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: ExecutionContext)
 
 @mainTemplate(
     title=pageTitle,
-    hidePtaAccountNav = true
```

Comment:
you prob need to add  showPtaAccountNav = false as  this is in the new main template - please check my branch payee-587

Reply:
sure thing, will update now!